src/book/02-system/03-tools/05-system-configuration.mdx

```diff
@@ -0,0 +1,64 @@
+---
+section: System
+chapter: Tools
+title: System Configuration
+description: A tool that sets up the robot.
+slug: /system/tools/system_configuration
+---
```

Comment:
```suggestion
slug: /system/tools/system_configuration
authors:
  - Cameron Murtagh (@CMurtagh-LGTM)
---
```

Reply:
Also do the same thing for Flashing a Robot page (can't comment on that part) and put yourself and Alex in as authors.